### DISCUSSION TITLE
When will the latest version be released on snap?

### DISCUSSION THREAD

**Discussion by Levi-Armstrong** [ros-industrial/ros_qtc_plugin#512] [Announcements]
@christian-rauch

**Comment by christian-rauch** [comment]
The latest Qt Creator 14 requires at least Qt 9.4.3. However, I cannot use a newer version than 6.2 in the snap (https://github.com/ros-industrial-snap/ros_qtc_plugin_snap/issues/3, https://forum.snapcraft.io/t/cannot-use-qt6-in-classic-core22-snap/34771) at the moment.

I thought about reusing the binary amd64 and arm64 distributions of Qt Creator instead of building it from source. But I haven't found the time to look into this properly and it's also not clear if this actually fixes the problem with the QPA plugins.
